data/mods/More_Survival_Tools/armor.json

```diff
@@ -263,5 +263,31 @@
         "bashing" : -5,
         "coverage" : 65,
         "material_thickness" : 1
+    },
+    {
+        "type" : "ARMOR",
+        "id" : "iggaak",
+        "name" : "makeshift sunglasses",
+        "name_plural" : "makeshift sunglasses",
+        "weight" : 70,
+        "color" : "light_gray",
+        "covers" : ["EYES"],
+        "to_hit" : -2,
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "Eyewear carved from from wood or bone, with only a thin slit to see out of. Helps reduce glare, traditionally used by the Inuit to protect against snow blindness.",
+        "price" : 4500,
+        "material" : ["bone", "null"],
+        "volume" : 1,
+        "cutting" : 0,
+        "warmth" : 10,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 15,
```

Comment:
For something that leaves just a thin slit to see through, this sounds way too low.